### PR TITLE
Added the ability to get Model via RouteModelBindings by passing the invite code

### PR DESCRIPTION
**Pull request by nklasio**
…Invite Code

This is required to fetch the Model via RouteModelBinging and passing only the code

### CODE REVIEW

File: src/Http/Models/Invite.php [hunk 1]
```diff
@@ -39,6 +39,14 @@ public function __construct(array $attributes = [])
         parent::__construct($attributes);
         $this->setTable(config('invite-codes.tables.invites_table'));
     }
+    
+    /**
+     * Sets 'code' as primary key for Route Model Bindings
```
Comment (mateusjunges) [inline src/Http/Models/Invite.php]: You should add a final dot here in order to the styleci checks to pass.

**Comment by mateusjunges** [comment]
Hi @nklasio, can you please fix [styleci][styleci] checks? I'm not able to merge it until those checks pass.

Also, i think it's good to add tests to this new feature.

[styleci]: https://github.styleci.io/analyses/PxBM3m

**Comment by nklasio** [comment]
I don't really get what else is wrong with the styling

File: src/Http/Models/Invite.php [hunk 2]
```diff
@@ -39,6 +39,14 @@ public function __construct(array $attributes = [])
         parent::__construct($attributes);
         $this->setTable(config('invite-codes.tables.invites_table'));
     }
+
```
Comment (mateusjunges) [inline src/Http/Models/Invite.php]: I think this line has blank spaces. Remove it and add a new line. It should fix the styleci checks.

Comment (nklasio) [reply src/Http/Models/Invite.php]: tried but github does not recognize it as a difference

Comment (mateusjunges) [reply src/Http/Models/Invite.php]: Add style ci to your repo. Then, you can analyze it there and merge a pull request in your branch fixing this issue.

**Comment by mateusjunges** [comment]
Good! Finally, can you add tests to it?

**Comment by nklasio** [comment]
I am not quite sure how to test for RouteModelBindings, but I can asure it works as I use it the same way in a project of mine

**Comment by mateusjunges** [comment]
Yeah i'm sure it works, but the tests ensure future update will not break it. I'll merge it as soon as i have time to write those tests. Thanks!

**Pull request merged**